index.bs

```diff
@@ -575,68 +575,182 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
+				readonly attribute PresentationStyle presentationStyle;
+				readonly attribute FrozenArray&lt;DOMString> types;
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+				Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+			dictionary ClipboardItemOptions {
+				PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
 
-		  readonly attribute FrozenArray&lt;DOMString> types;
+		<p><dfn for="ClipboardItem">Clipboard Item</dfn></p>
 
-		  Promise&lt;Blob> getType(DOMString type);
-		};
+		A [=clipboard item=] is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command.
+		For example, if a user copies a range of cells from a spreadsheet of a native application, it will result in one [=clipboard item=]. If a user copies a set of files from their desktop, that list of files will be represented by multiple [=clipboard item=]s.
+		Some platforms may support having more than one [=clipboard item=] at a time on the [=Clipboard=], while other platforms replace the previous [=clipboard item=] with the new one. 
+
+		A [=clipboard item=] has a <dfn>list of representations</dfn>, each <dfn>representation</dfn> with an associated <dfn>mime type</dfn> and <dfn>data</dfn>. In the example where the user copies a range of cells from a spreadsheet, it may be represented as an image (image/png), an HTML table (text/html), or plain text (text/plain).
+		Each of these [=mime type=]s describe a different [=representation=] of the same [=clipboard item=] at different levels of fidelity and make the [=clipboard item=] more consumable by target applications during paste.
+		Making the range of cells available as an image will allow the user to paste the cells into a photo editing app, while the text/plain format can be used by text editor apps.
+
+		A [=clipboard item=] can also optionally have a <dfn>presentation style</dfn> that helps distinguish whether apps "pasting" a [=clipboard item=] should insert the contents of an appropriate [=representation=] inline at the point of paste or if it should be treated as an attachment.
+
+		Apps that support pasting only a single [=clipboard item=] should use the first [=clipboard item=].
+		Apps that support pasting more than one [=clipboard item=] could, for example, provide a user interface that previews the contents of each [=clipboard item=] and allow the user to choose which one to paste.
```

Comment:
So, this is just a statement about usage of `ClipboardItem`/`ClipboardItem`s that is part of the processing model. I also mentioned this in the `Clipboard Interface` section, but added it here as well for completeness.

Reply:
> So, this is just a statement about usage of `ClipboardItem`/`ClipboardItem`s that is part of the processing model.

It's an example, it doesn't define the processing model.

 I also mentioned this in the `Clipboard Interface` section, but added it here as well for completeness.

Reply:
The core part of the sentence is that there could be more than one `clipboard item` which is relevant to the conceptual `clipboard item` object. The example is just a way to describe how one can use multiple `clipboard item`s.

Reply:
> The core part of the sentence is that there could be more than one `clipboard item` which is relevant to the conceptual `clipboard item` object.

Why is that relevant? There can be multiple `clipboard item` instances, but that doesn't affect the definition of the conceptual `clipboard item`.
The example itself is helpful. I think it would just be better placed in a note of the `Clipboard` interface, as mentioned above. Maybe I'm completely misunderstanding something...

Reply:
I don't think that particular sentence is related to definition of `clipboard item`. It just talks about how one can use multiple `clipboard item` objects. I don't really have a strong opinion so I can change it if others feel the same way.